src/components/Common/Error/Error.tsx

```
@@ -1,12 +1,10 @@
-import { ErrorCSS } from "./Error.styled";
+import { ErrorStyles } from "./Error.styled";
 
 export interface ErrorProps {
-  errorMessage: string;
+  errorMessage: string | null;
   isVisible: boolean;
 }
 
-function Error({ errorMessage, isVisible }: ErrorProps) {
-  return isVisible && <ErrorCSS>{errorMessage}</ErrorCSS>;
+export default function Error({ errorMessage, isVisible }: ErrorProps) {
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)